api/nodes/serializers.py

```diff
@@ -35,6 +35,45 @@
 from osf.utils import permissions as osf_permissions
 
 
+def get_institutions_to_add_remove(institutions, new_institutions):
+    diff = relationship_diff(
+        current_items={inst._id: inst for inst in institutions.all()},
+        new_items={inst['_id']: inst for inst in new_institutions}
+    )
+
+    insts_to_add = []
+    for inst_id in diff['add']:
+        inst = Institution.load(inst_id)
+        if not inst:
```

Comment:
Minor: You can do `Institution.objects.filter(_id=inst_id).exists()` here.

Reply:
Is that helpful/better? I need the inst below unless I refactor.